go.sum

```diff
@@ -88,6 +88,10 @@ github.com/hashicorp/yamux v0.1.1 h1:yrQxtgseBDrq9Y652vSRDvsKCJKOUD+GzTS4Y0Y8pvE
 github.com/hashicorp/yamux v0.1.1/go.mod h1:CtWFDAQgb7dxtzFs4tWbplKIe2jSi3+5vKbgIO0SLnQ=
 github.com/hookdeck/hookdeck-go-sdk v0.0.38 h1:lIrvEkK+I6VTjOM6FzQ+SKAZceKIOZgWaLnV2k8RLzY=
 github.com/hookdeck/hookdeck-go-sdk v0.0.38/go.mod h1:kfFn3/WEGcxuPkaaf8lAq9L+3nYg45GwGy4utH/Tnmg=
+github.com/hookdeck/hookdeck-go-sdk v0.1.0 h1:jhqIfd4NYBT99eQghbeUh/RabOZJ0aLtG0cU6wgNwxI=
+github.com/hookdeck/hookdeck-go-sdk v0.1.0/go.mod h1:kfFn3/WEGcxuPkaaf8lAq9L+3nYg45GwGy4utH/Tnmg=
+github.com/hookdeck/hookdeck-go-sdk v0.2.0 h1:IDY5dNiL4vKqyBAPXKa5CpKzl3kmHfObHnDGjJMvQUM=
+github.com/hookdeck/hookdeck-go-sdk v0.2.0/go.mod h1:kfFn3/WEGcxuPkaaf8lAq9L+3nYg45GwGy4utH/Tnmg=
```

Comment:
@alexluong I don't know anything about this file, but it seems strange that there are dependencies on different versions of the Hookdeck Go SDK.

Reply:
@leggetter I'm not a Go expert but my understanding is that Go keeps information of previous module versions there for some reason. I ran `go mod tidy` to clean this file a bit but I don't think it matters much in the grand scheme of thing.